js/web/docs/webnn-operators.md

```diff
@@ -41,6 +41,7 @@ operators and the supported opset domain/versions in **WebNN EP** by ONNX Runtim
 | GlobalLpPool| ai.onnx(7+) | l2Pool2d | ✗ | ✓ | Only supports 4-D input, 'p' value is 2 |
 | Greater | ai.onnx(7-8, 9-12, 13+) | greater | ✓ | ✓ | |
 | GreaterOrEqual | ai.onnx(12-15, 16+) | greaterOrEqual | ✓ | ✓ | |
+| GRU | ai.onnx(7-13, 14-21, 22+) | gru | ✓ | ✓ | Only supports 'layout' == 0. 'clip' is not supported. The activation functions in 'activations' must be one of 'Relu', 'Tanh', 'Sigmoid'. Forward and backward activations must be the same if bidirectional. 'sequence_lens' if present should be constant with values equal to the first dimension length of input 'X' |
```

Comment:
Do we need to mention `clip` given it's not in the MLRecurrentNetworkActivation list?

```
enum MLRecurrentNetworkActivation {
  "relu",
  "sigmoid",
  "tanh"
};
```

https://www.w3.org/TR/webnn/#api-mlgraphbuilder-gru

Reply:
In ORT, `clip` is applied to the input of activations. I think this is not the same as the activations in `MLRecurrentNetworkActivation` we defined. We just need to mention that `clip` of ORT is not supported here. WDYT?

Reply:
Clip isn't even a valid GRU activation though, according to ONNX?

```
Activation functions:

Relu(x) - max(0, x)
Tanh(x) - (1 - e^{-2x})/(1 + e^{-2x})
Sigmoid(x) - 1/(1 + e^{-x})
```

https://onnx.ai/onnx/operators/onnx__GRU.html

Reply:
Oh, do you mean the `clip` *attribute*, not the clip operator!?

Reply:
Yes, I mean the clip attribute.